web/src/app/util/ISOPickers.js

```diff
@@ -35,7 +35,7 @@ function useISOPicker(
     const dt = DateTime.fromFormat(input, format, { zone })
     if (dt.isValid) {
       if (type === 'time') {
-        return dtValue.set({
+        return DateTime.fromISO(value, { zone }).set({
```

Comment:
This won't work if `value` is null/empty.

This is the case when we have a time-only picker that starts with no value.

We can call that unsupported for now and make value required for `type="time"` instead of making assumptions here. We _could_ use the current timestamp, convert it to the zone, and then set hour & minute, but it might not work as expected -- and again we don't currently have a use-case for it.

Alternatively, add this towards the top, and then we can always do `dtValue.set` here.
```js
if (type === 'time' && !dtValue) throw new Error('initial value is required for type=time')
```

While we have a use-case for an empty date & time (or even an empty date) -- having an empty clock time is a little less meaningful and should probably be up to the parent to handle things like zone changes & DST by picking the appropriate reference date.